index.html

```diff
@@ -0,0 +1,754 @@
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Responsive Project</title>
+    <link rel="stylesheet" href="assets/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
+
+    <script src="https://kit.fontawesome.com/f58a22ad21.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+    <header>
+        <nav>
+            <div class="justFlex">
+                <div class="logo3">
+                    <img src="assets/images/logo3.PNG" alt="logo3">
+                </div>
+                <div class="screenMenu">
+                    <div class="menu flex-spaceBetween">
+                        <ul class="menu-item">
+                            <li><a href="#">News</a></li>
+                            <li><a href="#">Events</a></li>
+                            <li><a href="#">Business</a></li>
+                            <li><a href="#">AMAs</a></li>
+                            <li><a href="#">Spaces</a></li>
+                            <li><a href="#">Terms & Conditions</a></li>
+                        </ul>
+                        <ul class="menu-item">
+                            <li><a href="#">About</a></li>
+                            <li><a href="#">Advertise</a></li>
+                            <li><a href="#">Jobs</a></li>
+                            <li><a href="#">Contact</a></li>
+                        </ul>
+                    </div>
+                    <div class="menu2 flex-spaceBetween">
+                        <ul class="menu-item2">
+                            <li><a href="#">LATEST</a></li>
+                            <li><a href="#">HARD FORK</a></li>
+                            <li><a href="#">PLUGGED</a></li>
+                            <li><a href="#">README</a></li>
+                            <li><a href="#">GROWTH QUARTERS</a></li>
+                            <li><a href="#">NEURAL</a></li>
+                        </ul>
+                        <ul>
+                            <li><i class="fas fa-search"></i></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>    
+
+            <div class="menuMobile">
+                <div class="mobileMenu flex-spaceBetween">
+                    <div><img src="assets/images/logo2.PNG" alt="logo-image"></div>
+                    <div>
+                        <i class="fas fa-search"></i>
+                        <i class="fas fa-bars"></i>
+                    </div>
+                </div>
+                <div class="weHeard">
+                    We heard you like events and sweatpants so we made event you
+                    can join in your seatpants <i class="fas fa-long-arrow-alt-right"></i>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <section class="news-section flex">
+        <div class="news__article1">
```

Comment:
## This div element -> `news__article1` should have some space on its left in desktop mode.

Right now 👇
![Annotation 2020-06-04 230308](https://user-images.githubusercontent.com/57726348/83792210-58122980-a6b8-11ea-986c-ee888fa38a69.jpg)

It should be 👇
![Annotation 2020-06-04 230247](https://user-images.githubusercontent.com/57726348/83792236-619b9180-a6b8-11ea-8dfd-b6338a1c0d1b.jpg)